--- createdb.cpp	2016-04-26 21:23:57.000000000 +0100
+++ createdb.cpp	2016-05-18 14:19:52.238341898 +0100
@@ -2591,8 +2591,10 @@
   if(!RunQuery(sql)) {
     return false;
   }
-  sql="insert into LOGS (NAME,SERVICE,DESCRIPTION,ORIGIN_USER,ORIGIN_DATETIME)\
-      values (\"SAMPLE\",\"Production\",\"Sample Log\",\"user\",NOW())";
+  sql="insert into LOGS (NAME,SERVICE,DESCRIPTION,ORIGIN_USER,ORIGIN_DATETIME,\
+      LINK_DATETIME,MODIFIED_DATETIME,START_DATE,END_DATE)\
+      values (\"SAMPLE\",\"Production\",\"Sample Log\",\"user\",NOW(),\
+      NOW(),NOW(),NOW(),NOW())";
   if(!RunQuery(sql)) {
     return false;
   }
