--- opendb.cpp	2016-05-20 12:26:10.221807385 +0100
+++ opendb.cpp	2016-05-20 12:22:23.581023214 +0100
@@ -233,6 +233,18 @@
 	}
 	delete q;
 
+        //MySQL v5.7 compatibility fix
+        sql=QString("SET GLOBAL sql_mode=(SELECT REPLACE(@@sql_mode,'ONLY_FULL_GROUP_BY',''))");
+	q=new QSqlQuery(sql);
+	if(!q->isActive()) {   // Can't create DB.
+	  delete q;
+	  PrintError(QObject::tr("Unable to disable ONLY_FULL_GROUP_BY mode!"),
+		     interactive);
+	  db->removeDatabase("mysql");
+	  return false;
+	}
+	delete q;
+
         // Check if creating the database on the local machine or on a remote
         // server in the same subnet.  If creating on a remote server, set the
         // host portion of the MySQL user to the subnet that is common between
